azure-migrate/migrate-at-scale-vmware-agentles/Agentless VMware automation/readme.md

```diff
@@ -60,25 +73,54 @@ TARGET_MACHINE_NAME | Provide the name that the migrated VM should use in Azure.
+UPDATED_TAG_OPERATION | Specify the operation to be performed on the tag. Use "Merge" to mege the tag, "Delete" to delete the tag, or "Replace" to replace the tag.
+UPDATED_VM_TAG_KEY | Provide the name for the Virtual Machine Tag that needs to be updated for the VM to use in Azure. If there is more than one name, you can add multiple keys in the same column separated by comma.
+UPDATED_VM_TAG_VALUE | Provide the value for the Virtual Machine Tag that needs to be updated for the VM to use in Azure. If there are more than one value, you can add multiple values in the same column separated by comma. (The number of keys and values should be same.)
+UPDATED_VM_TAG_OPERATION | Specify the operation to be performed on the Virtual Machine tag. Use "Merge" to mege the tag, "Delete" to delete the tag, or "Replace" to replace the tag.
+UPDATED_DISK_TAG_KEY | Provide the name for the Disk Tag that needs to be updated for the VM to use in Azure. If there is more than one name, you can add multiple keys in the same column separated by comma.
+UPDATED_DISK_TAG_VALUE | Provide the value for the Disk Tag that needs to be updated for the VM to use in Azure. If there are more than one value, you can add multiple values in the same column separated by comma. (The number of names and values should be same.)
+UPDATED_DISK_TAG_OPERATION | Specify the operation to be performed on the Disk tag. Use "Merge" to mege the tag, "Delete" to delete the tag, or "Replace" to replace the tag.
+UPDATED_NIC_TAG_KEY | Provide the name for the NIC Tag that needs to be updated for the VM to use in Azure. If there is more than one name, you can add multiple keys in the same column separated by comma.
+UPDATED_NIC_TAG_VALUE | Provide the value for the NIC Tag that needs to be updated for the VM to use in Azure. If there are more than one value, you can add multiple values in the same column separated by comma. (The number of names and values should be same.)
+UPDATED_NIC_TAG_OPERATION | Specify the operation to be performed on the NIC tag. Use "Merge" to mege the tag, "Delete" to delete the tag, or "Replace" to replace the tag.
 UPDATED_TARGET_RESOURCE_GROUP_NAME | If you want to update the resource group to be used by the migrated VM in Azure, then specify the name of the Azure resource group, else leave blank. 
 UPDATED_TARGET_VNET_NAME | If you want to update the Virtual Network to be used by the migrated VM in Azure, then specify the name of the Azure Virtual Network, else leave blank.
 UPDATED_TARGET_MACHINE_NAME | If you want to update the name to be used by the migrated VM in Azure, then specify the new name to be used, else leave blank.
+UPDATED_TARGET_DISK_NAME | If you want to update the name to be used by the migrated VM in Azure for the disk, then specify the new name to be used, else leave blank.
```

Comment:
I think we update OS disk name in this case.
nit: If you want to update the disk name to be used by the migrated VM in Azure